package.json

```diff
@@ -12,6 +12,7 @@
         "prettier": "prettier --write .",
         "build": "npx tsc",
         "start": "node ./dist/index.js",
+        "start:prod": "node ./dist/index.js --prod",
```

Comment:
as discussed, we shouldn't have a prod/dev switch and different code paths based on where we're running.
We want to exercise the same code path regardless of the environment we run in. So let's remove the --prod switch and just always run `start`.
If we want to actually call okta or not for validation, we should instead change the OKTA url via environment variable.

I'll explain below